index.js

```diff
@@ -1,15 +1,17 @@
 const rollup = require('rollup');
 const path = require('path');
+const flattenArray = require('./lib/utils').flattenArray;
 
 const outputDefaults = {
     format: 'es'
 };
 
 function handleSuccess(responses, logger) {
-    const bundleCount = responses.reduce((accum, outputs) => {
-        return accum + outputs.reduce((accum, output) => {
-            return accum + Object.keys(output.output).length;
-        }, 0);
+    const flattenedResponses = flattenArray(responses);
+
+    const bundleCount = flattenedResponses.reduce((acc, response) => {
```

Comment:
Nice job. This is much easier to follow than nested reduces

Reply:
Agreed. Array.flat() will be supported by Node soon, and I won't need that flattenArray method any more.